app/models/basic_container_image.rb

```diff
@@ -0,0 +1,4 @@
+class BasicContainerImage < ContainerImage
```

Comment:
It is a class to represent a container image not from a specific provider. This is so that it will be easy to know what is the source of the image.